bagit.py

```diff
@@ -582,6 +586,37 @@ def files_to_be_fetched(self):
         for url, file_size, filename in self.fetch_entries():
             yield filename
 
+    def fetch_files_to_be_fetched(self):
+        """
+        Fetches files from the fetch.txt
+        """
+        urllib._urlopener = BagFetcherURLOpener # pylint: disable=protected-access
+        for url, expected_size, filename in self.fetch_entries():
+            expected_size = int(expected_size)  # FIXME should be int in the first place
```

Comment:
The spec also allows this to be `-` for unknown file sizes

Reply:
Fixed by a6aa50a